app/views/reservations/new.html.erb

```diff
@@ -1,7 +1,3 @@
 <div id='confirm-res-form'>
-	<% if @errors.empty? or (can? :override, :reservation_errors) %>
-	  <%= render :partial => "new_reservation" %>
-	<% else %>
-	  <%= render :partial => "new_request" %>
-	<% end %>
-</div>
\ No newline at end of file
+  <%= render partial: 'new_reservation' %>
```

Comment:
Yea fair enough. I feel like there might be a way to improve this but it's really not critical.